From f1420b2c5e27351956bf6d5a9177aedba84e1dad Mon Sep 17 00:00:00 2001
Message-Id: <f1420b2c5e27351956bf6d5a9177aedba84e1dad.1608221911.git.marksvc@gmail.com>
From: MarkS <marksvc@gmail.com>
Date: Thu, 17 Dec 2020 10:18:22 -0600
Subject: [PATCH] Add /app/lib to LD_LIBRARY_PATH

Change-Id: I69105393b28995c7d0c24a146bb733289ac256ac
---
 environ | 1 +
 1 file changed, 1 insertion(+)

diff --git a/environ b/environ
index 8d6097d8..14abe709 100644
--- a/fw/environ
+++ b/fw/environ
@@ -57,6 +57,7 @@ ${PATH}"
 	LD_LIBRARY_PATH="${BASE}/Output_${ARCH}/${BUILD}:\
 /usr/lib/fieldworks/lib:\
 ${COM}/build${ARCH}/lib:\
+/app/lib:\
 ${LD_LIBRARY_PATH}"
 fi
 # ensure we scan the default pkg-config directories (to pick up Geckofx for compiling)
-- 
2.25.1

