--- ./Makefile.in	2020-11-13 11:59:49.240008722 -0600
+++ ./Makefile.in	2020-11-13 12:02:38.784009504 -0600
@@ -43,12 +43,16 @@
 all: debug
 
 download_nuget:
+ifneq ($(EC_DOWNLOAD_DEPS), false)
 	build/multitry wget -O build/nuget.exe $(NUGET_URL) || build/multitry curl -o build/nuget.exe -L $(NUGET_URL)
+endif
 
 download_dependencies: download_nuget
+ifneq ($(EC_DOWNLOAD_DEPS), false)
 	. ./environ && build/multitry mono build/nuget.exe restore .nuget/packages.config -NonInteractive -PackagesDirectory "$(ROOTDIR)/packages"
 	cp -a packages/Geckofx45.64.Linux.45.0.21.0/build/Geckofx-Core.dll.config packages/Geckofx45.64.Linux.45.0.21.0/lib/net40
 	cp -a packages/Geckofx45.32.Linux.45.0.21.0/build/Geckofx-Core.dll.config packages/Geckofx45.32.Linux.45.0.21.0/lib/net40
+endif
 
 debug: download_dependencies
 	@cd build && ./msbuild.sh $(MONO) $(MONO_LDFLAGS) /property:FIELDWORKS=$(FIELDWORKS)
