From 87b0f9e1bf732b37b20729319ec07398b702fe48 Mon Sep 17 00:00:00 2001
Message-Id: <87b0f9e1bf732b37b20729319ec07398b702fe48.1605295194.git.marksvc@gmail.com>
From: MarkS <marksvc@gmail.com>
Date: Fri, 13 Nov 2020 13:19:35 -0600
Subject: [PATCH] Remove mistaken Xsl reference

---
 src/AIGuesserEC/AIGuesserEC-mono.csproj | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/AIGuesserEC/AIGuesserEC-mono.csproj b/src/AIGuesserEC/AIGuesserEC-mono.csproj
index 907cde2..edbe134 100644
--- a/src/AIGuesserEC/AIGuesserEC-mono.csproj
+++ b/src/AIGuesserEC/AIGuesserEC-mono.csproj
@@ -80,7 +80,6 @@
     <Reference Include="System.Management" />
     <Reference Include="System.Windows.Forms" />
     <Reference Include="System.Xml" />
-    <Reference Include="System.Xml.Xsl" />
     <Reference Include="System.Xml.Linq">
     </Reference>
   </ItemGroup>
-- 
2.25.1

